app/views.py

```diff
@@ -40,8 +40,9 @@ def home():
 @app.route('/dashboard')
 def dashboard():
```

Comment:
Using the same form won't work for both the Class and Quiz creation. You need to create 2 forms similar to my Registration.